lib/supply/client.rb

```diff
@@ -161,6 +161,45 @@ def listing_for_language(language)
       end
     end
 
+    # Get a list of all apks verion codes - returns the list of version codes
+    def apks_version_codes
+      ensure_active_edit!
+
+      result = api_client.execute(
+        api_method: android_publisher.edits.apks.list,
+        parameters: {
+          'editId' => current_edit.data.id,
+          'packageName' => current_package_name
+        },
+        authorization: auth_client
+      )
+
+      raise result.error_message.red if result.error? && result.status != 404
+
+      return result.data.apks.collect(&:versionCode)
+    end
+
+    # Get a list of all apk listings (changelogs) - returns the list
+    def apk_listings(apk_version_code)
+      ensure_active_edit!
+
+      result = api_client.execute(
+        api_method: android_publisher.edits.apklistings.list,
+        parameters: {
+          'apkVersionCode' => apk_version_code,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.